app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,44 @@
+class PasswordResetsController < ApplicationController
+  before_action :find_by_reset_token, only: [:edit, :update]
+  def new
+  end
+
+  def create
+    user = User.find_by_email(params[:email])
+    if user
+      user.send_password_reset 
+      @reset_url = edit_password_reset_url(user.password_reset_token)
+      # TODO: send email
+      redirect_to root_url, :notice => "Email sent with password reset instructions."
```

Comment:
no need to use a hash rocket in here. Also, maybe use `_path` as you're still inside your application